### PR TITLE
Added back/cancel links to edit and cancel to new collections pages.

### LINKED ISSUE #128: Provide back button/link to collection list page on collection edit page

### PR DESCRIPTION
**Pull request by justinlittman**
closes #128

### CODE REVIEW

File: app/views/collections/_form.html.erb [hunk 1]
```diff
@@ -16,6 +16,7 @@
         ] %>
       <%= form.input :active %>
       <%= form.button :submit, class: 'btn-primary' %>
+      <%= link_to 'Cancel', collections_path, html_options={style: "margin-left: 25px"} %>
```
Comment (jcoyne) [inline app/views/collections/_form.html.erb]: This doesn't seem like valid Ruby.  Do we have a test for this file?

Comment (justinlittman) [reply app/views/collections/_form.html.erb]: Test added.

Comment (jcoyne) [reply app/views/collections/_form.html.erb]: I don't understand why you are assigning a hash to the local variable named `html_options`

Comment (justinlittman) [reply app/views/collections/_form.html.erb]: It's part of the method signature: https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to

Comment (jcoyne) [reply app/views/collections/_form.html.erb]: But when you call a method you don't pass in the method signature.  In a signature that means, the third parameter is called html_options, and if it is not provided, here is the default.  When you're invoking you don't want the `html_options=`

Comment (justinlittman) [reply app/views/collections/_form.html.erb]: Removed `html_options`.

File: app/views/collections/_form.html.erb [hunk 2]
```diff
@@ -16,6 +16,7 @@
         ] %>
       <%= form.input :active %>
       <%= form.button :submit, class: 'btn-primary' %>
+      <%= link_to 'Cancel', collections_path, {style: "margin-left: 25px"} %>
```
Comment (jcoyne) [inline app/views/collections/_form.html.erb]: If a hash is the final argument, you can omit the `{ }`

File: spec/requests/update_collection.rb [hunk 3]
```diff
@@ -1,15 +0,0 @@
-# frozen_string_literal: true
```
Comment (jcoyne) [inline spec/requests/update_collection.rb]: Why is this file removed? Should this be a different PR?

Comment (mjgiarlo) [reply spec/requests/update_collection.rb]: This has been removed in master so shouldn't need to be in this branch.

Comment (jcoyne) [reply spec/requests/update_collection.rb]: @justinlittman please rebase then
